src/test/java/com/example/java/test/junior/developer/JavaTestJuniorDeveloperApplicationTests.java

```diff
@@ -1,15 +1,91 @@
 package com.example.java.test.junior.developer;
 
+import static org.junit.jupiter.api.Assertions.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.Arrays;
+
+import com.example.java.test.junior.developer.model.Language;
+import com.example.java.test.junior.developer.repository.LanguageRepository;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.ActiveProfiles;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
 
-@ActiveProfiles("test")
 @SpringBootTest
+@AutoConfigureMockMvc
```

Comment:
In your new class please use `@WebMvcTest` - and not `@SpringBootTest` - you will not need `@AutoConfigureMockMvc`.